addons/l10n_account_edi_ubl_cii_tests/tests/common.py

```diff
@@ -234,9 +234,9 @@ def _assert_invoice_attachment(self, attachment, xpaths, expected_file_path):
         xml_content = base64.b64decode(attachment.with_context(bin_size=False).datas)
         xml_etree = self.get_xml_tree_from_string(xml_content)
 
-        expected_file = get_resource_path(self.test_module, 'tests/test_files', expected_file_path)
-        self.assertTrue(expected_file, "expected file not found")
-        expected_etree = etree.parse(expected_file).getroot()
+        expected_file_full_path = get_resource_path(self.test_module, 'tests/test_files', expected_file_path)
+        self.assertTrue(expected_file_full_path, "expected file not found")
+        expected_etree = etree.parse(expected_file_full_path).getroot()
```

Comment:
useless diff

Reply:
I disagree: I put a wrong and misleading variable name and it needs to be corrected, except if you consider it's not an issue to have wrong misleading variable name.

Reply:
IRL: we decided to put it in another commit to not pollute the original commit with the diff